script.js

```diff
@@ -303,3 +308,36 @@ function appendStory(story, board, storyUrl, statusBadge, estimate) {
     "</div>"
   );
 }
+
+function initializeDragAndDrop () {
+  $(".todo, .in-progress, .ready-for-review, .done").sortable({
+    connectWith: ".todo, .in-progress, .ready-for-review, .done"
+  });
+
+  $(".js-board").on("sortstop", function(event, ui) {
+      updatePivotal(ui);
+  });
+} //end initializeDragAndDrop
+function updatePivotal(ui) {
+    let id = ui.item.data("id");
+    console.log(ui.item);
+    let newStatus = ui.item.closest(".js-board").data("status");
+    let beforeId = ui.item.prev().data("id");
+    let urlId = "/" + ui.item.data("id");
+    if(beforeId === undefined) beforeId = null;
```

Comment:
missing space before conditional. Should be:

```
if (beforeId === undefined) beforeId = null;
```

Reply:
edited